get-started/index.md

```diff
@@ -12,23 +12,23 @@ Cryostat {{ site.data.versions.cryostat.version }}
 {:toc}
 
 ## [Installing Cryostat Operator](#installing-cryostat-operator)
-Follow the steps below to install the **Cryostat Operator** via [OperatorHub](https://operatorhub.io/operator/cryostat-operator).
+Follow the steps below to install the **Cryostat Operator** via [**OperatorHub**](https://operatorhub.io/operator/cryostat-operator).
 
 ### [Install cert-manager](#install-cert-manager)
 The **Cryostat Operator** requires [cert-manager](https://cert-manager.io/) to run.
 If not already installed in your cluster, please
 [install](https://cert-manager.io/docs/installation/) it using your preferred method.
-Once installed, proceed with the Operator installation steps below.
+Once installed, proceed with the **Operator** installation steps below.
 
 <span style="color:red">**Warning**:</span> Although it is possible to [disable cert-manager integration](https://github.com/cryostatio/cryostat-operator/blob/{{site.data.versions.cryostat.release-branch}}/docs/config.md#disabling-cert-manager-integration), it is NOT recommended to
 do so unless cert-manager is unavailable AND one of the following applies to you:
 - You have another solution for encrypting traffic
 - You trust everything running in the same cluster where the **Cryostat Operator** is deployed
 
 ### [Install via OperatorHub](#install-via-operatorhub)
-See below for a summary of the installation steps from the **Cryostat Operator** page on [OperatorHub](https://operatorhub.io/cryostat-operator). For more details, visit [Installing the Cryostat Operator from OperatorHub](https://developers.redhat.com/articles/2022/01/20/install-cryostat-operator-kubernetes-operatorhubio#).
+See below for a summary of the installation steps from the **Cryostat Operator** page on [**OperatorHub**](https://operatorhub.io/cryostat-operator). For more details, visit [Installing the **Cryostat Operator** from OperatorHub](https://developers.redhat.com/articles/2022/01/20/install-cryostat-operator-kubernetes-operatorhubio#).
 
-1. Install the Operator Lifecycle Manager **(OLM)**:
+1. Install the **Operator** Lifecycle Manager **(OLM)**:
```

Comment:
I think the rest of this document uses "**Operator**" as shorthand to refer to our specific **Cryostat Operator**. In this case, "Operator" is part of the name **Operator Lifecycle Manager** - I'm not sure we want/need to bold that, since **OLM** is already bolded and is more commonly referred to in the rest of the document.